src/Services/Air/Air.js

```diff
@@ -15,6 +15,20 @@ module.exports = (settings) => {
       return service.searchLowFares(options);
     },
 
+    fareRules(options) {
```

Comment:
I've researched this topic before implementing:
Fare rules are linked to a fare basis and route used, and these are the params.

Because fetching fare rules is part of an AirPriceReq, passengers are a required param; passing different passengers creates "virtual" fare quotes for these passengers, applying the fare basis to these passenger types and returning complete **pricing information** about a future, not yet existing booking. In terms of uAPI each **air:AirPricingSolution** / **air:AirPricingInfo** has a **air:FareInfo** calculation; and each **air:FareRule** information is linked to each **air:FareInfo** by reference.

Fare rules text, on the other hand (short and long), **contains information about applying the farebasis to all PTCs** and **are linked to the fare basis itself**. For example, a long description fetched using only ADT, contains text about how this fare is applied to CNN and INF.

We can have different fare bases applied for different passengers (known as \*MULTI\* pricing options as seen in terminal), but currently `fareBasisCode` is a field in `segments`, so basically  if we want to factor out this simplification, we have to move `fareBasisCode` from `segments` into `bookingComponents`, and this is out of scope for current implementation.

There'a s also problem, that sometimes two identical fare information sets are returned, because the same `fareBasisCode` is used for forward and return legs.